src/styles/mixins/_typography.scss

```diff
@@ -16,7 +16,15 @@
     @include font-aliasing;
 
     font-weight: normal;
-    font-size: 13px;
+    font-size: $bdl-fontSize;
+}
+
+@mixin bdl-fonts-dejaBlue {
```

Comment:
--dejaBlue if you want to keep it as a modifier pattern like you did with the variable name